schema/deploy/shipping/views.sql

```diff
@@ -3160,29 +3169,43 @@ create or replace view shipping.__uw_priority_queue_v1 as (
             case
                 when daily_symptoms then 1
 
-                /* Why 2 days after attestation to exposures?  Our questions
-                 * about exposure ask about the past 24 hours.  We don't want
+                /* Why 2 days after attestation to exposures? We don't want
                  * to offer testing too early after exposure and risk false
-                 * negatives.  Per Trevor, we expect that a viral load high
+                 * negatives. Per Trevor, we expect that a viral load high
                  * enough to detect with PCR and be contagious is generally
                  * centered on day 3 after exposure, though some will be
                  * earlier and some later.  By offering testing at day 2, some
                  * people will get tested that same day (day 2), most will
                  * (hopefully) get tested the next day (day 3), and some others
                  * by the day after that (day 4). This puts results being known
                  * at days 3–5.
-                 *    -trs, 19 Oct 2020
+                 * -trs, 19 Oct 2020
+
+                 * The `yes_now` option replaced `yes` in Winter Quarter 2021,
+                 * with the wording for `yes_now` being `Yes, I was exposed to COVID-19
+                 * in the last 48 hours`, but we kept both in the query so that the query
+                 * would work before and after the option change.
                  */
-                when daily_exposure_known_pos and age(__uw_encounters.encountered) >= '2 days' then 1
-                when daily_exposure and age(__uw_encounters.encountered) >= '2 days' then 4
-                else null
+                when daily_exposure_known_pos in ('yes', 'yes_now') and age(__uw_encounters.encountered) >= '2 days' then 1
+                when daily_exposure and age(__uw_encounters.encountered) >= '2 days' then 1
+
+                /* The `yes_late` option takes care of the 48 hours. The option label is
+                 * `Yes, I was exposed to COVID-19 more than 48 hours ago`
+                 */
+                when daily_exposure_known_pos = 'yes_late' then 1
```

Comment:
Just want to verify the `yes_now` and `yes_late` options are set in stone. I don't see them in the current HCT REDCap project.

Reply:
From Jessica just now: "I am submitting changes today to update the exposure questions."
She confirmed those two new 'yes' values.